CHANGELOG.rst

```diff
@@ -28,6 +28,10 @@ Fixed
 
   Contributed by @guzzijones
 
+Removed
+~~~~~~~~
+* Removed check-licence script since it's obsolete
```

Comment:
Can you please follow the same Changelog format with (change type) and PR number? See other examples below.
Feel free to include your name or username as a contributor.

Besides of that, looks good and st2-packages CI should report green status after we merged the https://github.com/StackStorm/st2-packages/pull/677.